packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -74,14 +74,20 @@ const UserAvatar = (
 };
 UserAvatar.displayName = 'UserAvatar';
 
-const stateReducer: DownshiftProps<{}>['stateReducer'] = (_state, changes) => {
+function getIsFocusedElementInMenu() {
+  return Boolean(
+    document?.activeElement?.attributes['data-user-settings-menu']
```

Comment:
Doesn't the document always exist?

Reply:
Unless the dog eats it 😆 

Yeah I think that's right, no need to do the check.

Reply:
```suggestion
    document.activeElement?.attributes['data-user-settings-menu']
```

Here so you can commit them from GH :)

Reply:
Already did the exact same thing 😄 🏎️